marc.go

```diff
@@ -195,6 +196,7 @@ func (m *MarcIterator) Err() error {
 
 func (m *MarcIterator) scanIntoRecord(bytes []byte) (Record, error) {
 	rec := Record{}
+	rec.Data = append([]byte(nil), bytes...)
```

Comment:
Nice. This should indeed allow for a mode in Mario that saves error records optionally.